R/servers.R

```diff
@@ -296,11 +296,12 @@ clientForAccount <- function(account) {
 }
 
 ensureConnectServerUrl <- function(url) {
+  # strip trailing /
+  url <- gsub("/$", "", url)
```

Comment:
This function needs to return a URL that can be concatenated with sub-paths like `/content`, which is why `/__api__` isn't given a trailing slash.

The bit that smooshes it all together:
https://github.com/rstudio/rsconnect/blob/5e0189d2e52c6708af942a600b115530c0094b1f/R/http.R#L276-L277